packages/rx_bloc_cli/analysis_options.yaml

```diff
@@ -1,5 +1,3 @@
-include: package:pedantic/analysis_options.yaml
```

Comment:
@DDavidPrime, I believe here we need to include: package:flutter_lints/flutter.yaml